### ISSUE TITLE
Visual Studio Deployment does not take InstanceCount into account

### ISSUE THREAD

**Issue by senj** [Azure/service-fabric-issues#46]
Hey guys,

I deployed a guest executable from visual studio to my dev SF cluster.
I noticed that that my application will always be on one node only, no matter what the InstanceCount value says. If I delete the deployed application and recreate one via the service fabric explorer, the instance count is correct.
Is this by design?

**Comment by vturecek** [comment]
The Visual Studio project templates set up an overridable instance count parameter that can be changed per deployment environment. That parameter value has the following flow:

Application\PublishProfiles\Cloud.xml | Local.xml -->
Application\ApplicationManifest.xml <Parameters ..  /> -->
Application\ApplicationManifest.xml <DefaultServices><Your Service .. />

If you deploy through Visual Studio, by default, it will take whatever is set in those PublishProfiles (depending on where you are publishing - local, cloud, or you can add others) and override all the way down into the Default Service settings.

The PublishProfile XML files are purely a Visual Studio thing. If you deploy through SFX, or anywhere else, it won't use those PublishProfile XML files; instead, you have to provide those parameters yourself, otherwise, it will use the default values in the <Parameters .. /> block in ApplicationManifest.xml.

So, long story short, check the values in your Local.xml/Cloud.xml files in Visual Studio.

**Comment by senj** [comment]
Great, thank you!
